build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -1,28 +1,20 @@
-<!--
-  1. Change "category" below to one of:
-       ads-analytics
-       dynamic-content
-       layout
-       media
-       presentation
-       social
-  2. Remove any of the "formats" that don't apply.
-     You can also add the "ads" and "stories" formats if they apply.
-  3. And remove this comment! (no empty lines before "---")
--->
-
 ---
-
-\$category: presentation
+\$category@: presentation
```

Comment:
Is the `@` character here intentional? If this is how frontmatter categories are supposed to be specified, I believe we'll need to change the existing uses.

Reply:
It is intentional. All other `.md` files seem to follow the format, and the generation of the file seemed to work okay for me